app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="header">
```

Comment:
Since this really is a navigation, what do you think about using more semantic markup?
i.e.
```
<nav>
  <ul>
    <li class="header-item><%= link_to "Home page", root_path %>
    ...
  </ul>
</nav>
```

Reply:
👌